Documentation/ExtensionArchitecture/HowTo/Localization/TypoScript.rst

```diff
@@ -56,7 +56,9 @@ Overriding translations in non-Extbase plugins does not work out of the box.
 
 ..  attention::
     Setting :php:`_LOCAL_LANG` has no effect if the ViewHelper
-    :html:`<f:translate>` is used outside of the Extbase context.
+    :html:`<f:translate>` is used outside of the Extbase context
```

Comment:
I wonder if we should negate this whole section, like:

```
Setting :php:`_LOCAL_LANG` can only have an effect if:
* :html:`<f:translate>` is used inside Extbase context (not a Fluid StandaloneView),
* OR the ViewHelper's :html:`extensionName` attribute is set to the proper extension key,
* OR the ViewHelper's :html:`key` attribute uses the `LLL:EXT:extensionkey/Path/To/locallang.xlf:languagekey` syntax.
```

Reply:
I still don't see how this is supposed to work outside of Extbase. Which part of code would apply the `_LOCAL_LANG` settings then? If I am for example in the context of a FLUIDTEMPLATE, how would this work?

Reply:
So we should then say extbase and fluid standalone?

Reply:
Hhhhm... maybe?! I don't know. If "Non-Extbase" is the same as "Fluid Standalone", then yes. I don't know if there is another scenario to use _LOCAL_LANG that is something else than "Fluid Standalone" but not "Extbase"...?

Reply:
Also this does not fit with line 55 here.

Reply:
Do you have an example where this is actually working? Or some code reference?

Reply:
Minimal example:

page = PAGE
page.10 = FLUIDTEMPLATE
page.10.template = TEXT
page.10.template.value (

  Hello <f:translate key="onlineDocumentation" extensionName="backend" />
  
  Hello <f:translate key="LLL:EXT:backend/Resources/Private/locallang.xlf:onlineDocumentation" />
  
)

Reply:
maybe even the example is worth documenting. I didn't dig to the very end why you have to prefix the extension with tx_, but not using tx_ will not work

Reply:
Wow, very interesting! Yes I think it would be worth documenting this!

Reply:
I like the current approach, that's very helpful! Thanks so much!

Reply:
I like it too, thanks a lot